README.md

```diff
@@ -15,6 +15,19 @@ See the [getting started doc](docs/getting_started.md) on support on how to get
 After installation, refer back to the below sections for supported scenarios and how to tweak the kraken config to load them on your cluster
 
 
+### Setting up infrastructure dependencies
+Kraken indexes the metrics specified in the profile into Elasticsearch and visualizes them in Grafana in addition to leveraging Cerberus for understanding the health of the kubernetes/OpenShift cluster under test. More information on the features is documented below. The infrastruture pieces can be easily installed, uninstalled by running:
+
+```
+$ cd kraken
+$ podman-compose up or $ docker-compose up    # Spins up the containers specified in the docker-compose.yml file present in the run directory
```

Comment:
Might be worth mentioning here that this command will continuously run/don't end it till after kraken execution

Reply:
Fixed it, thanks.